plugins/wordpress-rule-exclusions-before.conf

```diff
@@ -108,6 +108,24 @@ SecRule REQUEST_FILENAME "@endsWith /wp-admin/comment.php" \
     ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:newcomment_author,\
     ver:'wordpress-rule-exclusions-plugin/1.0.1'"
 
+# Replying to a comment
+SecRule REQUEST_FILENAME "@endsWith /wp-admin/admin-ajax.php" \
+    "id:9507132,\
+    phase:2,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'wordpress-rule-exclusions-plugin/1.0.1',\
+    chain"
+    SecRule ARGS:action "@streq replyto-comment" \
+        "t:none,\
+        chain"
+        SecRule &ARGS:action "@eq 1" \
+            "t:none,\
+            ctl:ruleRemoveTargetById=941100;ARGS:content,\
+            ctl:ruleRemoveTargetById=941160;ARGS:content,\
+            ctl:ruleRemoveTargetById=941310;ARGS:content"
```

Comment:
Is this free form text, shouldn't CRS be disabled for this input in that case?

Reply:
Yes, it is but this kind of FP was, obviously, seen for the first time in a whole history of our WordPress exclusion rules so let's start with this.

Reply:
Looks like this already was the case in the past, is this a new comment endpoint in WordPress 6.5?
https://github.com/coreruleset/wordpress-rule-exclusions-plugin/blob/c9cfcce66c45c5a3a5a1c990e2c80e6544fc13fc/plugins/wordpress-rule-exclusions-before.conf#L94

Reply:
Probably no, i was able to find a `replyto-comment` action also in vanilla WordPress 3.3 source code.

Reply:
I mean, what would happen if someone was running a technical blog and somebody posted code in the comments asking for help from the author? I think if we know it's reasonable for something like that to happen, then we should be proactive with false positives rather than trying to avoid false negatives. It just saves time for everybody.